bin/check-haproxy.rb

```diff
@@ -181,6 +191,8 @@ def acquire_services #rubocop:disable all
         # #YELLOW
       end.reject do |svc| # rubocop: disable MultilineBlockChain
         %w(FRONTEND BACKEND).include?(svc[:svname])
+      end.reject do |svc| # rubocop: disable MultilineBlockChain
+        config[:ignore_ini] && svc[:check_status] == 'INI'
```

Comment:
This would be more clear as

```ruby
haproxy_stats.select do |svc|
  next if %w(FRONTEND BACKEND).include?(svc[:svname]) || config[:ignore_ini] && svc[:check_status] == 'INI'
  svc[:pxname] =~ regexp
end
```